.github/pull_request_template.md

```diff
@@ -1,12 +1,47 @@
-# Release Version
-Current version: **X.Y.Z**
+<!-- Don't forget to provide a concise title  -->
 
-New version: **X.Y.Z**
+## Description
 
-This is a SemVer *MAJOR/MINOR/PATCH* release because...
+<!-- Provide a high level description of changes to the software -->
 
-# Summary of changes
-**TODO**: Please summarise the changes.
+- Current Version: x.y.z
+- New Version: x.y.z
 
-# Evidence of manual testing (if applicable)
-**TODO**: Please provide evidence of testing where appropriate.
+## Summary of changes
+
+<!-- Provide a bulleted list of changes e.g new features, bug fixes, performance 
+improvements etc -->
+
+- change 1
+- change 2
+
+## Testing Evidence
+
+### Test Environment
+
+<!-- Describe the environment used to test e.g. OS/ browser/ version -->
+
+These tests were conducted using the local-mdis-docker repo to set up a virtual
+SInfoE
+
+### Tests Conducted  
+
+<!-- Describe/ list the tests you ran to verify functionality -->
+
+### Results
+
+<!-- Insert an image, or summarise the testing results  -->
+
+## Checklist
+
+Please confirm:
+
+- [ ] Code follows the guidelines
```

Comment:
We already require the code to pass out CI pipelines, wouldn't that capture this requirement implicitly, or are there other guidelines we're following as well? 🤔